test/acceptance/view_benchmark_graphs_test.rb

```diff
@@ -90,6 +141,35 @@ class ViewBenchmarkGraphsTest < AcceptanceTest
     end
   end
 
+  test "User should be able to view long running benchmark graphs after they select different number of benchmark 
+  runs, when they go back and forward in the broswer window" do
+
+    javascript_driver = Capybara.javascript_driver
+    default_driver = Capybara.current_driver
+    Capybara.javascript_driver = :selenium
+    Capybara.current_driver = :selenium
+
+    benchmark_type = benchmark_types(:array_count)
+    BenchmarkRun.stub_const(:PAGINATE_COUNT, [1, 3]) do
+      BenchmarkRun.stub_const(:DEFAULT_PAGINATE_COUNT, 1) do
+        visit "/ruby/ruby/commits?result_type=#{benchmark_type.category}"
+
+        assert assert_selector(".highcharts-markers path", count: 1)
+        select 3, from: "benchmark_run_display_count"
+        assert assert_selector(".highcharts-markers path", count: 3)
+        
+        select 1, from: "benchmark_run_display_count"
+        page.evaluate_script('window.history.back()')
+        assert page.has_selector? '#benchmark_run_display_count', text: 3
+        page.evaluate_script('window.history.forward()')
+        assert assert_selector(".highcharts-markers path", count: 1)
+        assert page.has_selector? '#benchmark_run_display_count', text: 1
+
```

Comment:
:scissors: I should do up a style guide. My preference is to not leave a blank line at the end.